openmoltools/utils.py

```diff
@@ -479,3 +485,54 @@ def get_checkmol_descriptors( molecule_filename, executable_name = 'checkmol' ):
  
     return descriptors
 
+def amber_to_gromacs( molecule_name, in_prmtop, in_crd, out_top = None, out_gro = None): 
+    """Use ParmEd to convert AMBER prmtop and crd files to GROMACS format.
+
+    Requires
+    --------
+    Currently requires gromacs_support branch of ParmEd
```

Comment:
The gromacs_support branch has been merged into Parmed, right?  So this requirement can be changed to a version number requirement?